Rakefile

```diff
@@ -1,17 +1,19 @@
 # frozen_string_literal: true
 
 require 'English'
-SWIFTLINT_VERSION = '0.27.0'
-XCODE_WORKSPACE = 'WordPress.xcworkspace'
-XCODE_SCHEME = 'WordPress'
-XCODE_CONFIGURATION = 'Debug'
-
 require 'fileutils'
 require 'tmpdir'
 require 'rake/clean'
 require 'yaml'
 require 'digest'
 
+RUBY_REPO_VERSION = File.read('./.ruby-version').strip
+SWIFTLINT_VERSION = '0.27.0'
```

Comment:
Note: not directly related to this PR, but [latest SwiftLint](https://github.com/realm/SwiftLint/releases) being `0.47.1`, we might want to update it via a separate PR at some point 🙃